experimental/add_to_app/ios_using_plugin/IOSUsingPlugin/AppDelegate.swift

```diff
@@ -16,7 +16,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Instantiate Flutter engine
```

Comment:
(can't comment that far but want to comment on line 13)

Can't we do the same thing here? You're not constructing engine with anything special from the app delegate. Just lazy var the engine and pass it a default initializer like

```swift
lazy var flutterEngine: FlutterEngine = {FlutterEngine(name:..., project:...)}
```